homeassistant/components/motioneye/__init__.py

```diff
@@ -228,28 +237,31 @@ def _build_url(
     if entry.options.get(CONF_WEBHOOK_SET, DEFAULT_WEBHOOK_SET):
         url = async_generate_motioneye_webhook(hass, entry.data[CONF_WEBHOOK_ID])
 
-        if _set_webhook(
-            _build_url(
-                device,
-                url,
-                EVENT_MOTION_DETECTED,
-                EVENT_MOTION_DETECTED_KEYS,
-            ),
-            KEY_WEB_HOOK_NOTIFICATIONS_URL,
-            KEY_WEB_HOOK_NOTIFICATIONS_HTTP_METHOD,
-            KEY_WEB_HOOK_NOTIFICATIONS_ENABLED,
-            camera,
-        ) | _set_webhook(
-            _build_url(
-                device,
-                url,
-                EVENT_FILE_STORED,
-                EVENT_FILE_STORED_KEYS,
-            ),
-            KEY_WEB_HOOK_STORAGE_URL,
-            KEY_WEB_HOOK_STORAGE_HTTP_METHOD,
-            KEY_WEB_HOOK_STORAGE_ENABLED,
-            camera,
+        if url and (
+            _set_webhook(
+                _build_url(
+                    device,
+                    url,
+                    EVENT_MOTION_DETECTED,
+                    EVENT_MOTION_DETECTED_KEYS,
+                ),
+                KEY_WEB_HOOK_NOTIFICATIONS_URL,
+                KEY_WEB_HOOK_NOTIFICATIONS_HTTP_METHOD,
+                KEY_WEB_HOOK_NOTIFICATIONS_ENABLED,
+                camera,
+            )
+            | _set_webhook(
```

Comment:
`_set_webhook` returns a boolean. Now because in Python boolean is a subclass of int this is working, but that's not the correct logical operator.
```suggestion
            or _set_webhook(
```